src/SimpleThings/EntityAudit/DependencyInjection/SimpleThingsEntityAuditExtension.php

```diff
@@ -58,5 +56,13 @@ public function load(array $configs, ContainerBuilder $container)
         if (true === $config['listener']['current_username']) {
             $loader->load('current_username.xml');
         }
+
+        if ($config['audit_connection']) {
+            $container->getDefinition('implethings_entityaudit.config')
+                ->addMethodCall('setAuditConnection', array(
+                        new Reference($config['audit_connection'])
```

Comment:
Hm, it works when I use:
`new Reference('doctrine.dbal.' . $config['audit_connection'] . '_connection')`